app/src/main/java/org/feature/fox/coffee_counter/util/Event.kt

```diff
@@ -0,0 +1,17 @@
+package org.feature.fox.coffee_counter.util
+
+open class Event<out T>(private val content: T) {
```

Comment:
Is currently unused and can be removed.

Reply:
Of course the approval is only if the changes requested in Discord are applied.

Reply:
> Is currently unused and can be removed.

Should be resolved with 1dbf6c6dbcd7daaa8e88db7e8eff37beb968d496



> Of course the approval is only if the changes requested in Discord are applied.

Changes should be applied with 82f488247fadeff832541905042f82b929470491